q2_vsearch/tests/test_join_pairs.py

```diff
@@ -258,58 +258,6 @@ def test_join_pairs_alt_maxee(self):
         # confirm altered parameter was passed to vsearch
         self.assertTrue('--fastq_maxee 25.0' in ' '.join(cmd))
 
-    def test_join_pairs_alt_qmin(self):
-        with redirected_stdio(stderr=os.devnull):
-            cmd, obs = _join_pairs_w_command_output(
-                self.input_seqs, qmin=-1)
-
-        # sanity check the output
-        self._test_manifest(obs)
-        output_fastqs = list(obs.sequences.iter_views(FastqGzFormat))
-        self.assertEqual(len(output_fastqs), 3)
-
-        # confirm altered parameter was passed to vsearch
-        self.assertTrue('--fastq_qmin -1' in ' '.join(cmd))
-
-    def test_join_pairs_alt_qminout(self):
-        with redirected_stdio(stderr=os.devnull):
-            cmd, obs = _join_pairs_w_command_output(
-                self.input_seqs, qminout=-1)
```

Comment:
With newer versions of vsearch this test now produces the following error:

```bash
Fatal error: Sum of arguments to --fastq_asciiout and --fastq_qminout must be no less than 33
```

@torognes - when I tried adding the `--fastq_asciiout` flag to our base command (to address the error above) I observed the following error:

```bash
Fatal error: Invalid options to command fastq_mergepairs
Invalid option(s): --fastq_asciiout
```

Given that, it seems like specifying a non-zero value for `--fastq_qminout` isn't possible? Is this a mistake on our end?

Reply:
Yeah, that's what I was trying to point out, the error says you need to use the `--fastq_asciiout` flag, but that flag isn't valid with this command.

Reply:
Sorry for the late reply.

The argument to the `--qminout` option cannot be negative (at least as long as `--fastq_asciiout` is set to 33). It is 0 by default.

The `--fastq_asciiout` option is 33 by default and can only be changed to 64 when using the commands for converting between SFF and FASTQ, between old and new FASTQ formats, or from FASTA to a fake FASTQ format. It specifies the ascii character that correspond to a quality score of 0 in the FASTQ output files. There is another option named `--fastq_ascii` that specifies this for the FASTQ input files.

You may set `--qminout` to a higher value, e.g. 7, to clamp all FASTQ quality values to be at least 7.